### ISSUE TITLE
SyntaxError: illegal character; error loading preferences dialog for GSConnect

### ISSUE THREAD

**Issue by kayfay** [andyholmes/gnome-shell-extension-gsconnect#608]
**Describe the bug**
Installed from Github, doens't load, wouldn't install through gnome tweak tool either.

**Steps To Reproduce:**

Install from github
open gnome tweak tool
GSConnect: There was an error loading the preferences dialog for GSConnect:
```
SyntaxError: illegal character

Stack trace:
  @/home/atools/.local/share/gnome-shell/extensions/gsconnect@andyholmes.github.io/prefs.js:16
  Application<._getExtensionPrefsModule@resource:///org/gnome/shell/extensionPrefs/main.js:74
  wrapper@resource:///org/gnome/gjs/modules/lang.js:178
  Application<._selectExtension@resource:///org/gnome/shell/extensionPrefs/main.js:89
  wrapper@resource:///org/gnome/gjs/modules/lang.js:178
  Application<._onCommandLine@resource:///org/gnome/shell/extensionPrefs/main.js:239
  wrapper@resource:///org/gnome/gjs/modules/lang.js:178
  main@resource:///org/gnome/shell/extensionPrefs/main.js:377
  @<main>:1
  
```

**Expected behavior**

expected to do something

**Screenshots**

![](https://i.imgur.com/e38wZHn.png)

**Support Log**

Please generate a support log ([Instructions](wiki/Help#generate-support-log)) and paste any messages related to this issue between the two ``` lines below.

```
I Can't Open the Tool to Generate
```

**System Details (please complete the following information):**

 - **GSConnect version:** [e.g. 20, 24, ... (displayed in the GSConnect "About" window)]
The 8/11 version.
   - **Installed from:** [e.g. GNOME Extensions Website, GitHub, Package Manager, ...]
GitHub
 - **GNOME/Shell version:** [e.g. 3.30, 3.28, ...]
3.22.3
 - **Distro/Release:** [e.g. Ubuntu 18.04, Fedora 29, Arch, ...]
Debian Stretch
**GSConnect environment (if applicable):**

 - **Paired Device(s):** [e.g. Galaxy Note 8, Pixel 2, Honor 9, ...]
 - **KDE Connect app version:** [e.g. 1.12.7]
 - **Plugin(s):** [if the issue only occurs when using certain plugin(s)]
N/A

**Additional Notes:**

**Comment by andyholmes** [comment]
Hi, sorry but GSConnect currently only support GNOME Shell 3.28-3.32. I would recommend just using KDE Connect on GNOME Shell 3.22.

**Comment by isaaclw** [comment]
Ah, that makes sense. I didn't see any documentation of that on either extensions.gnome.org or the wiki. Do you think you could add that?

**Comment by andyholmes** [comment]
On extensions.gnome.org there is just no version available for 3.22. In the wiki it's documented on the [Installation](https://github.com/andyholmes/gnome-shell-extension-gsconnect/wiki/Installation#dependencies) page that we only support the latest version of GNOME.
